editoast/src/main.rs

```diff
@@ -412,6 +422,37 @@ async fn add_electrical_profile_set(
     Ok(())
 }
 
+async fn electrical_profile_set_list(
+    pg_config: PostgresConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let mut conn = PgConnection::establish(&pg_config.url()).await?;
+    let electrical_profile_sets = ElectricalProfileSet::list_light(&mut conn).await.unwrap();
+    println!("Electrical profile sets:\nID - Name");
+    for electrical_profile_set in electrical_profile_sets {
+        println!(
+            "{:<2} - {}",
+            electrical_profile_set.id.unwrap(),
+            electrical_profile_set.name.unwrap()
+        );
+    }
+    Ok(())
+}
+
+async fn electrical_profile_set_delete(
+    args: DeleteProfileSetArgs,
+    pg_config: PostgresConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let manager = ConnectionManager::<PgConnection>::new(pg_config.url());
+    let pool = Data::new(Pool::builder(manager).max_size(1).build().unwrap());
+    for profile_set_id in args.profile_set_ids {
+        ElectricalProfileSet::delete(pool.clone(), profile_set_id)
```

Comment:
That's a detail, but if the profile set doesn't exist, it will still print the same message. Since `delete` returns a `bool` it may be useful to print a different message

Reply:
Definitely a good idea